src/SeleniumLibrary/keywords/element.py

```diff
@@ -56,35 +56,54 @@ def element_should_contain(self, locator, expected, message=None):
         The ``message`` argument can be used to override the default error
         message.
 
+        The ``ignore_case`` argument can be set to True to compare case
+        insensitive, default is False.
+
         Use `Element Text Should Be` if you want to match the exact text,
         not a substring.
         """
-        actual = self.find_element(locator).text
+        actual = actual_before = self.find_element(locator).text
+        expected_before = expected
+
+        if is_truthy(ignore_case):
+            actual = actual.lower()
+            expected = expected.lower()
+
         if expected not in actual:
             if is_noney(message):
                 message = "Element '%s' should have contained text '%s' but "\
-                          "its text was '%s'." % (locator, expected, actual)
+                          "its text was '%s'." % (locator, expected_before, actual_before)
+
             raise AssertionError(message)
-        self.info("Element '%s' contains text '%s'." % (locator, expected))
+        self.info("Element '%s' contains text '%s'." % (locator, expected_before))
 
     @keyword
-    def element_should_not_contain(self, locator, expected, message=None):
+    def element_should_not_contain(self, locator, expected, message=None, ignore_case=False ):
         """Verifies that element ``locator`` does not contains text ``expected``.
 
         See the `Locating elements` section for details about the locator
         syntax.
 
         The ``message`` argument can be used to override the default error
         message.
+
+        The ``ignore_case`` argument can be set to True to compare case
+        insensitive, default is False.
         """
         actual = self.find_element(locator).text
+        expected_before = expected
+
```

Comment:
We are not in favor of code blocks. Please remove the empty line from this line and from line 99.